packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -314,7 +314,7 @@ module.exports = {
     'operator-linebreak': 'off',
 
     // enforce padding within blocks
-    'padded-blocks': ['error', 'never'],
+    'padded-blocks': ['error', { blocks: 'never', classes: 'never', switches: 'never' }],
```

Comment:
i mean that "classes" and "switches" should be set to null, or omitted - and that in a second commit/PR, they should be set to "never".